scrapper/discudemy.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import time
+from bs4 import BeautifulSoup
+# local import
+import fetch
+import util
+
+
+async def get(session, url="https://www.discudemy.com/all/1"):
+    link_card = set()
+    links_udemy = set()
+    start = time.time()
+    num_calls = 1
+    links_second_screen = set()
+
+    cont = await fetch.get_all(session, url)
+    if cont:
+        for html in cont:
+            soup = BeautifulSoup(html, "html.parser")
+            card = soup.find_all(
+                "a", class_="card-header")
+            link_card = [course.get("href") for course in card]
+
+    cont = await fetch.get_all(session, link_card)
+    num_calls += len(link_card)
+    if cont:
```

Comment:
Each of these 3 sets of `if cont:` could be broken out into a private helper function. This would lower the complexity of this function.